vernac/vernacprop.ml

```diff
@@ -0,0 +1,53 @@
+(************************************************************************)
```

Comment:
Yeah not sure, maybe if vernac_classifier needs access to the document then it belongs in stm.